api/src/main/java/uk/gov/hmcts/payment/api/configuration/security/PaymentAuthCheckerConfiguration.java

```diff
@@ -15,7 +15,7 @@
 import java.util.regex.Pattern;
 
 @Configuration
-public class AuthCheckerConfiguration {
+public class PaymentAuthCheckerConfiguration {
```

Comment:
Upgrade resulted in some additional checks on app startup that required this being named differently from the other `AuthCheckerConfiguration` from the S2S libs.